kong/plugins/httplog/schema.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  logging_url = { required = true, type = "string" }
```

Comment:
There are ways of validating the URL, for example with a `reggae` field, like in the DAO's `apis.lua`